### PR TITLE
Feat(web): Remove `Accordion` Header active background change

### PR DESCRIPTION
**Pull request by pavelklibani**
### Issue reference

[Komponenta Accordion | úpravy chování na mobilu](https://jira.almacareer.tech/browse/DS-1365)

### CODE REVIEW

File: packages/web/src/scss/components/Accordion/_Accordion.scss
Comment (adamkudrna) [inline packages/web/src/scss/components/Accordion/_Accordion.scss:1]: Please improve the commit message so it says what has been changed and why, as per our usual standards 🙂.

**Review by adamkudrna — CHANGES_REQUESTED** [review]
I tested it in mobile Safari and it works as expected, i.e. the header background doesn't flicker on interaction.

File: packages/web/src/scss/components/Accordion/_Accordion.scss
Comment (pavelklibani) [reply packages/web/src/scss/components/Accordion/_Accordion.scss:1]: commit message updated

**Review by crishpeen — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]

**Review by literat — APPROVED** [review]
LGTM 👍

**Pull request merged**